apps/web/graphql/users/logic.ts

```diff
@@ -101,13 +101,12 @@ export const updateUser = async (userData: any, ctx: GQLContext) => {
     checkIfAuthenticated(ctx);
     const { id } = userData;
 
-    const hasPermissionToManageUser = checkPermission(ctx.user.permissions, [
-        permissions.manageUsers,
-    ]);
-    if (!hasPermissionToManageUser) {
-        if (id !== ctx.user._id.toString()) {
-            throw new Error(responses.action_not_allowed);
-        }
+    if (!checkPermission(ctx.user.permissions, [permissions.manageUsers])) {
+        throw new Error(responses.action_not_allowed);
+    }
+
+    if (id === ctx.user._id.toString()) {
```

Comment:
In this case the user won't be able to edit other properties of their profile like their name and bio etc. We only want to restrict the user from altering their `permissions` and `active`, other things like name, bio, tags should be editable via themselves.